open_fdd/desktop/drivers/weather_driver.py

```
@@ -0,0 +1,30 @@
+    end = datetime.now(timezone.utc).replace(minute=0, second=0, microsecond=0)
+    start = end - timedelta(days=max(1, int(days_back)))
+    rng = pd.date_range(start=start, end=end, freq="1h", tz="UTC")
+    frame = pd.DataFrame(
+        {
+            "timestamp": rng,
+            "outside_air_temp_c": [12.0] * len(rng),
+            "outside_air_rh_pct": [55.0] * len(rng),
+        }
+    )
+    store.write_frame(source="weather", site_id=site_id, frame=frame)
+    return WeatherFetchResult(rows=int(len(frame.index)))
```

Ruff (0.15.12)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)